sha2/sha2.c

```diff
@@ -32,7 +32,12 @@
  * $Id: sha2.c,v 1.1 2001/11/08 00:01:51 adg Exp adg $
  */
 
+#if defined(_WIN32) || defined(_WIN64)
+#include "..\tinydtls.h"
```

Comment:
Is this required?

`target_include_directories(tinydtls PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})` should make it obsolete.